lib/inbox.rb

```diff
@@ -305,17 +318,23 @@ def deltas(cursor, exclude_types=[], expanded_view=false)
       end
     end
 
-    def delta_stream(cursor, exclude_types=[], timeout=0, expanded_view=false)
+    def delta_stream(cursor, exclude_types=[], include_types=[], timeout=0, expanded_view=false)
```

Comment:
I think better to use hash as parameter here e.g.:  {include: :messages, exclude: :files, :threads, expanded_view: true}